flox-bash/lib/commands/environment.sh

```diff
@@ -55,9 +57,12 @@ function floxList() {
 	trace "$@"
 	local environment="$1"; shift
 	local system="$1"; shift
+	parseNixArgs "$@" && set -- "${_cmdArgs[@]}"
+	parseFloxFlakeArgs "$@" && set -- "${_cmdArgs[@]}"
 	# set $branchName,$floxNixDir,$environment{Name,Alias,Owner,System,BaseDir,BinDir,ParentDir,MetaDir}
 	eval $(decodeEnvironment "$environment")
 	parseNixArgs "$@" && set -- "${_cmdArgs[@]}"
+	parseFloxFlakeArgs "$@" && set -- "${_cmdArgs[@]}"
 	local -a invocation=("$@")
```

Comment:
doe we really need the parse*Args twice here? anything that decodeEnvironment would add to it?

Reply:
Nope that's a mistake ... well spotted, will remove the second instance.